cms-adapter/src/Rsbc.Dmf.CaseManagement/Manager/CaseManager.cs

```diff
@@ -856,9 +889,10 @@ public async Task<CaseDetail> GetCaseDetail(string caseId)
                 var fetchedCase = dynamicsContext.incidents
                     .Expand(c => c.dfp_DriverId)
                     .Expand(c => c.dfp_DriverId.dfp_PersonId)
+                    .Expand(c => c.dfp_incident_dfp_knownmedicalcondition)
```

Comment:
note to self: low risk but still worth regression testing, I'd say regression test the portals would suffice